src/onig-scanner.cc

```diff
@@ -58,8 +58,8 @@ OnigScanner::OnigScanner(Local<Array> sources) {
   regExps.resize(length);
 
   for (int i = 0; i < length; i++) {
-    String::Utf8Value utf8Value(sources->Get(i));
-    regExps[i] = shared_ptr<OnigRegExp>(new OnigRegExp(string(*utf8Value)));
+    Local<String> source = Local<String>::Cast(sources->Get(i));
+    regExps[i] = shared_ptr<OnigRegExp>(new OnigRegExp(OnigString(source)));
```

Comment:
Yeah, that makes perfect sense. Can always be done later-ish. Other that this one comment, I reckon this PR looks stellar :sparkles: